docs/src/main/asciidoc/security-openid-connect-dev-services.adoc

```diff
@@ -156,6 +158,19 @@ image::dev-ui-keycloak-sign-in-to-service.png[alt=Dev UI OpenId Connect Keycloak
 
 Set a relative service endpoint path, click on `Sign In To Service` and you will be redirected to Keycloak to enter a username and password in a new browser tab and get a response from the Quarkus application.
 
+Note that in this case `Dev UI` does not really enrich a dev experience since it is a Quarkus OIDC `web-app` application which controls the authorization code flow and acquires the tokens.
```

Comment:
```suggestion
Note that in this case Dev UI does not really enrich a dev experience since it is a Quarkus OIDC `web-app` application which controls the authorization code flow and acquires the tokens.
```